common/api/core-frontend.api.md

```diff
@@ -2434,6 +2353,12 @@ export enum CoordinateLockOverrides {
     None = 0
 }
 
+// @internal (undocumented)
+export class CoordinatesUtils {

```

Comment:
nit but this class adds nothing vs just defining a free function called `deflateCoordinates`.

Reply:
done